src/vic3/tables/triggers.rs

```diff
@@ -426,7 +426,7 @@ const TRIGGER: &[(Scopes, &str, Trigger)] = &[
     (Scopes::Country, "has_wasted_construction", Boolean),
     (Scopes::None, "hidden_trigger", Control),
     (Scopes::Country, "highest_secession_progress", CompareValue),
-    (Scopes::InterestGroup, "ig_approval", CompareChoice(APPROVALS)),
+    (Scopes::InterestGroup, "ig_approval", CompareApproval),
```

Comment:
Mhmmm I originally made the change to move away from specialization. I think a `CompareNumberChoice` would be more appropriate for code reuse later.

Reply:
It's a good point but we do have only one user for it :)

I'll change it though

Reply:
Done